### DISCUSSION TITLE
Add a option to store dashboard/datasource config directly in git

### DISCUSSION THREAD

**Discussion by treilly94** [grafana/grafana#43864] [General]
It would be really useful if there was a option to have changes to dashboards/datasources form git commits that could be pushed to a git repo rather than being stored in the database. 
I've seen this functionality in other tools such as [dataiku](https://doc.dataiku.com/dss/latest/collaboration/version-control.html) and have found it really useful. It could also help address some of the other issues raised around versioning of the dashboards like https://github.com/grafana/grafana/issues/9235

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
